syntax = "proto3";

package pb;

import "code.proto";

option go_package = ".;protocol";

service Greeter {
  rpc SayHello(CSReqHello) returns (CSResHello) {}
  rpc SendChat(CSReqSendChat) returns (CSResSendChat) {}
}

message CSReqHello {
  string name = 1;
}

message CSResHello {
  string message = 1;
}

message CSReqSendChat {
  string msg = 1;
}

message CSResSendChat {
  Code ret = 1;
}
